base/remote/tests/testthat/test.remote.execute.cmd.R

```diff
@@ -0,0 +1,16 @@
+context("Testing remote.execute.cmd() for different possible inputs")
```

Comment:
The testthat team [no longer recommends using `context`](https://testthat.r-lib.org/reference/context.html) -- they recommend (and I agree) having one context per file and picking filenames that convey it well.